activerecord/test/cases/quoting_test.rb

```diff
@@ -72,7 +72,7 @@ def test_quoted_time_utc
           t = Time.now.change(usec: 0)
 
           expected = t.change(year: 2000, month: 1, day: 1)
-          expected = expected.getutc.to_s(:db).sub("2000-01-01 ", "")
+          expected = expected.getutc.to_s(:db).sub(/\A\d\d\d\d-\d\d-\d\d /, "")
```

Comment:
Can you change it to this:

``` ruby
expected = t.getutc.change(year: 2000, month: 1, day: 1)
expected = expected.to_s(:db).sub("2000-01-01 ", "")
```

It makes the expected value less permissive and the reduces the chance of it passing when it should fail - thanks!

Reply:
@pixeltrix Fixed. Thanks for the clarifying this.